orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -5900,13 +5910,11 @@ def find_input_node_type(model, arg):
     gathergrad_input_optypes = [find_input_node_type(training_model, arg) for arg in gathergrad_node.input]
     if test_op == "Add" or test_op == "Mul" or test_op == "Sub":
         assert test_op in gathergrad_input_optypes
-    elif test_op == "MatMul":
+    else:
         if case == 0:
+            assert test_op in gathergrad_input_optypes
+        else:
             assert "ATen" in gathergrad_input_optypes
```

Comment:
Aten will be always be in gathergrad_input_optypes? 

```suggestion
            assert "ATen" in gathergrad_input_optypes
```

Reply:
Yep, only 'MatMul' case 1 and 'ReduceMean' case 1 will go into this else, and "Aten" should be in gathergrad_input_optypes.